### PR TITLE
Fill in missing Safari version numbers...

### PR DESCRIPTION
**Pull request by mdittmer**
... using latest data from [Web API Confluence Dashboard](https://web-confluence.appspot.com/#!/catalog)

This change explicitly avoids touching any existing numeric versions. To reproduce automated PR creation process (up to showing diff on working tree):

```bash
git clone https://github.com/mdn/browser-compat-data.git

mkdir mdn-confluence
cd mdn-confluence
git init
git remote add origin https://github.com/mdittmer/mdn-confluence.git
git fetch origin ed6c1b467e21e7ce7a44e53603fa13dac3027f91
git reset --hard FETCH_HEAD
npm install

rm -rf node_modules/browser-compat-data/*
cp -r ../browser-compat-data/* node_modules/browser-compat-data/

node main/generate.es6.js --browsers=safari --output-dir=../browser-compat-data --fill-only

cd ../browser-compat-data
git diff
```

### CODE REVIEW

File: api/Animation.json [hunk 1]
```diff
@@ -342,7 +342,8 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "10",
+              "version_removed": "11"
```
Comment (Elchi3) [inline api/Animation.json]: Was Animation.effect really removed?

File: api/Animation.json [hunk 2]
```diff
@@ -1327,7 +1328,8 @@
...
```
Comment (Elchi3) [inline api/Animation.json]: Was Animation.timeline really removed?

File: api/File.json [hunk 3]
```diff
@@ -193,7 +193,8 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "6",
+              "version_removed": "10"
```
Comment (Elchi3) [inline api/File.json]: Was File.lastModifiedDate really removed?

File: api/HTMLBaseFontElement.json [hunk 4]
```diff
@@ -82,7 +82,8 @@
               "version_added": false
             },
             "safari": {
-              "version_added": null
+              "version_added": "6",
+              "version_removed": "10"
```
Comment (Elchi3) [inline api/HTMLBaseFontElement.json]: If this and the other properties were removed, was the whole HTMLBaseFontElement interface removed maybe?

File: api/NodeFilter.json [hunk 5]
```diff
@@ -82,7 +82,8 @@
               "version_added": "9"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "3",
+              "version_removed": "10"
```
Comment (Elchi3) [inline api/NodeFilter.json]: Was NodeFilter.acceptNode really removed?

File: api/NodeIterator.json [hunk 6]
```diff
@@ -84,7 +84,8 @@
               "version_added": "9"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "3",
+              "version_removed": "10"
```
Comment (Elchi3) [inline api/NodeIterator.json]: Was NodeIterator.expandEntityReferences really removed?

File: api/URLSearchParams.json [hunk 7]
```diff
@@ -85,7 +85,8 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
+              "version_removed": "10"
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.append really removed?

File: api/URLSearchParams.json [hunk 8]
```diff
@@ -136,7 +137,8 @@
...
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.delete really removed?

File: api/URLSearchParams.json [hunk 9]
```diff
@@ -187,7 +189,8 @@
...
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.get really removed?

File: api/URLSearchParams.json [hunk 10]
```diff
@@ -238,7 +241,8 @@
...
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.getAll really removed?

File: api/URLSearchParams.json [hunk 11]
```diff
@@ -289,7 +293,8 @@
...
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.has really removed?

File: api/URLSearchParams.json [hunk 12]
```diff
@@ -340,7 +345,8 @@
...
```
Comment (Elchi3) [inline api/URLSearchParams.json]: Was URLSearchParams.set really removed?

File: javascript/builtins/Promise.json [hunk 13]
```diff
@@ -249,7 +249,8 @@
                 "version_added": "50"
               },
               "safari": {
-                "version_added": false
+                "version_added": true,
+                "version_removed": "8"
```
Comment (Elchi3) [inline javascript/builtins/Promise.json]: Promise.finally is relatively new. I don't think it was in Safari before 8 and removed in 8 also seems off.

File: javascript/builtins/String.json [hunk 14]
```diff
@@ -1154,7 +1154,8 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": true
+                "version_added": true,
+                "version_removed": "9"
```
Comment (Elchi3) [inline javascript/builtins/String.json]: String.length was probably not removed

File: javascript/builtins/Symbol.json [hunk 15]
```diff
@@ -600,7 +600,8 @@
                 "version_added": "38"
               },
               "safari": {
-                "version_added": "10"
+                "version_added": "10",
+                "version_removed": "9"
```
Comment (Elchi3) [inline javascript/builtins/Symbol.json]: Symbol.species was probably not removed. Also version_removed can never be older than version_added. I will add an issue about this, so our tests catch this, too.

File: javascript/builtins/Symbol.json [hunk 16]
```diff
@@ -855,7 +856,8 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": true
+                "version_added": true,
+                "version_removed": "9"
```
Comment (Elchi3) [inline javascript/builtins/Symbol.json]: Symbol.toStringTag was probably not removed?

File: api/Animation.json (hunk 1 above)
Comment (gsnedders) [reply api/Animation.json]: In latest Safari TP (so beyond 11), the Animation interface isn't exposed.

**Comment by Elchi3** [comment]
No activity here, so I'm closing this now. Please re-open if you want to come back to it.

**Pull request closed without merging**